styles.css

```diff
@@ -0,0 +1,136 @@
+:root {
+  --color-primary: hsl(0, 0%, 20%);
+  --color-primary-light: hsl(0, 0%, 30%);
+  --color-accent: #ff6b00;
+  --nav-height: 60px;
+}
+
+body {
```

Comment:
- [ ] [OPTIONAL] You could improve your website more by adding custom fonts. You could make your choice from the open-source [google fonts ](https://fonts.google.com/)library